components/ai_chat/core/browser/conversation_handler.h

```diff
@@ -28,6 +41,12 @@
 #include "url/gurl.h"
 
 class AIChatUIBrowserTest;
+namespace mojo {
+template <typename Interface>
+class PendingRemote;
```

Comment:
100% optional but I'd almost prefer we just import the header

Reply:
I think I will let only this one behind, unless Brian dislikes it.